frontend/app/mocks/node.ts

```diff
@@ -86,6 +107,22 @@ const handlers = [
     return HttpResponse.text(null, { status: 204 });
   }),
 
+  /**
+   * Handler for GET request to retrieve address by id
+   */
+  http.get('https://api.example.com/users/:userId/addresses/:addressId', ({ params }) => {
```

Comment:
You must use the `userId` param to fetch the user first then use that user to fetch his related address.

Reply:
Fixed in 2d834de935ac55c6e3fd5b53090e72574032ff3b